blocks/cards/cards.css

```diff
@@ -25,3 +25,28 @@
   aspect-ratio: 4 / 3;
   object-fit: cover;
 }
+
+/* Day 2 – Part 1  Assignment #5 start  */
+.anchor-anchorLink {
+  display: flex; 
+  justify-content: center; 
+  align-items: center; 
+  padding-left: 15px; 
+  background-color: #f0f0f0; 
+  border: 1px solid #ccc; 
+  border-radius: 4px; 
+  text-decoration: none; 
+  color: #333; 
+  font-size: 16px; 
+  width: 200px;
+  margin-left: 25px;
+}
+
+.anchor-anchorLink:hover {
+  background-color: #e0e0e0; 
+  color: #000; 
```

Comment:
Try to use the color variables instead of hardcoding here.

Reply:
I have changed the CSS.